rapier/rules-update-2020-07.md

```diff
@@ -0,0 +1,63 @@
+--- 
+title: Updates from latest Society Rules - July 2020
+layout: default 
+--- 
+ 
+* TOC 
+{:toc} 
+ 
+**Author**: Master Brian de Moray, KRM 
+ 
+**Last Modified**: {{ page.last_modified_at | date: '%Y-%m-%d' }}
+ 
+**Revision**: {{ site.git.last_commit.short_sha }}
+
+# Rubber-tipped Spears
+
+Atlantia now allows for the use of rubber-tipped spears in both rapier and C&T combat.  Rapier Spears are specifically precluded from use in light rapier.  For the foreseeable future, all Rapier Spear activities must be actively marshaled by a rapier marshal with a Rapier Spear authorization.
+
+All those marshaling Rapier Spear activity should record and submit detailed feedback in the Event Report as to the efficacy, safety, and any issues raised during Rapier Spear activities.
+
+The marshal-in-charge of an activity may limit Rapier Spears on a per-activity basis as long as the limitation is widely announced.  As an example, after bringing all participants together, announce: "For the next melee, you may not use Rapier Spears."

```

Comment:
Are spears default and not-spears special?

Reply:
Being the default would spur adootion.  However,For spear activities to happen, there must be an authorized Marshall. Perhaps as part of the announcement, they should indicate if the issue was that no spear authorized marshalls were available.

Reply:
My intent is to remove obstacles towards approved equipment by default.

If a marshal-in-charge wishes to intentionally focus their activity, this is acceptable as long as it's announced.  We see this in practice already, such as "armored as worn" tournaments.

Reply:
Specifically called out when spears are default-on vs not default on.